config/config.yaml

```diff
@@ -28,6 +27,12 @@ defaults:
     etcd:
       kvName: arohcp-etcd-{{ .ctx.regionShort }}
       kvSoftDelete: true
+    istio:
+      url: "https://github.com/istio/istio/releases/download"
```

Comment:
Is there ever a case where we will download from some other GitHub release page? I don't think so - you are likely good to remove this as a parameter and simply hard-code it into the script.